.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Welcome to the OpenTracing Go repo! 👋🎉
+
+- Please be respectful and considerate of others when commenting on issues.
+- Please search for existing issues in order to ensure we don't have duplicate bugs/feature requests.
+- Please don't ask questions here. If you have any question head to our gitter chat https://gitter.im/opentracing/public
```

Comment:
rephrase less confrontationally.

If you have a question please ask it on our Gitter chat https://gitter.im/opentracing/public instead of creating an issue.